mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_Mobile.java

```diff
@@ -118,6 +118,7 @@ public class IITC_Mobile extends AppCompatActivity
     private String debugInputStore = "";
     private Map<String, String> mAllowedHostnames = new HashMap<>();
     private Set<String> mInternalHostnames = new HashSet<>();
+    private final Pattern mGoogleHostnamePattern = Pattern.compile("google\\.(com\\.|co\\.).\\w+$");
```

Comment:
```suggestion
    private final Pattern mGoogleHostnamePattern = Pattern.compile("google\\.(com\\.|co\\.)?\\w+$");
```
If the match should work as mentioned in comment (for google.com?.*), The pattern should look like this, right? 🤔

Reply:
fixup half fixed this, (`?.` instead of `.`), it's `?` now
I kept the `(com\.|co\.)?` for readability (instead of `(com?\.)?`, should I use `(com?\.)?` instead ?

Reply:
Imho it's good like this.
I'd maybe prefer having the dot on the other side like this: ...google(\\.com|\\.co)?\\.\\w+
For me it reads a bit better, but it's really just a cosmetics 🤷